src/test/kotlin/com/defenseunicorns/koci/RegistryTest.kt

```diff
@@ -292,6 +294,17 @@ class RegistryTest {
         }
     }
 
+    @Test
+    @EnabledIfSystemProperty(named = "TESTS_WITH_EXTERNAL_SERVICES", matches = "true")
+    fun `pull and remove gradle`() = runTest(timeout = 10.minutes) {
```

Comment:
Need to add `storage.Remove` and ensure cleanup has happened so shared storage is left in a clean state